### PR TITLE
DevWs for SHINA SYSTEM containing containing SiHAS USM-300-Z

### PR DESCRIPTION
**Pull request by shinasys**

### CODE REVIEW

**Comment by PKacprowiczS** [comment]
@shinasys is this the same device as in #46732?

**Comment by shinasys** [comment]
> @shinasys is this the same device as in #46732?

This #46732 is a test version. I will delete it in the future.

File: devicetypes/shinasys/sihas-usm-300-z.src/sihas-usm-300-z.groovy [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+		capability "Battery"
+		capability "Motion Sensor"
+		capability "Temperature Measurement"
+		capability "Illuminance Measurement"
+		capability "Relative Humidity Measurement"
+        capability "Health Check"
+        capability "Sensor"
+        capability "Refresh"
+        
+        fingerprint profileId: "0104", deviceId: "0001", inClusters: "0000,0001,0003,0004,0020,0400,0402,0406,0500,0B05", outClusters: "0003,0004,0019", manufacturer: "ShinaSystem", model: "USM-300Z", deviceJoinName: "SiHAS USM-300Z"
```
Comment (PKacprowiczS) [inline devicetypes/shinasys/sihas-usm-300-z.src/sihas-usm-300-z.groovy]: Change `deviceJoinName` to `SiHAS  Multipurpose Sensor`

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]
Whole code is hard to read. 
First thing you should take care of, is to change approach how you parse Zigbee messages. Please have a look at e.g. Zigbee Motion/Temp/Humidity Sensor as an example.
Additionally, pay attention to code indication, spacing. 
We won't accept such poorly written code, as it is now.

File: devicetypes/shinasys/sihas-usm-300-z.src/sihas-usm-300-z.groovy (hunk 1 above)
Comment (shinasys) [reply devicetypes/shinasys/sihas-usm-300-z.src/sihas-usm-300-z.groovy]: In the method you requested, I will do it again later.

**Comment by KKlimczukS** [comment]
@shinasys Please, could You share any update on this pull request ?

**Pull request closed without merging**